components/style/gecko_bindings/test_bindings.rs

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
```

Comment:
We don't want to include it in `gecko_bindings`, we want to include (`include!()`, not `pub mod`) it in `tests/unit/stylo`. The glue functions don't get defined till later. I plan on doing this myself later.